google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java

```diff
@@ -212,13 +230,32 @@ Logging getLogging() {
     if (logging == null) {
       synchronized (this) {
         if (logging == null) {
-          logging = LoggingOptions.getDefaultInstance().getService();
+          logging = createLoggingOptions().getService();
         }
       }
     }
     return logging;
   }
 
+  /** Creates the {@link LoggingOptions} to use for this {@link LoggingAppender}. */
+  LoggingOptions createLoggingOptions() {
```

Comment:
Can we do a similar approach as LoggingHandler (saving loggingOptions as a variable)?

Reply:
Changed to store the LoggingOptions as a variable.